client/landing/stepper/declarative-flow/internals/steps-repository/create-site/index.tsx

```diff
@@ -84,14 +81,17 @@ const CreateSite: Step = function CreateSite( { navigation, flow, data } ) {
 		planCartItem,
 		selectedSiteTitle,
 		productCartItems,
+		siteUrl,
```

Comment:
Let's add the `progress` variable here since it's referenced at the bottom of the file.

```suggestion
		siteUrl,
		progress,
```